crocket/src/main/java/com/crocket/model/entity/Hoop.java

```diff
@@ -105,13 +106,20 @@ private boolean passedThrough(Ball ball) {
     }
 
     public Hitbox getLeftHitbox() {
-        return new Hitbox(leftHitbox.getWidth(), leftHitbox.getHeight(), leftHitbox.getxPosition(),
-                leftHitbox.getyPosition());
+        return this.leftHitbox;
```

Comment:
I'm not sure which way is the best, but I think the old way was to prevent anyone except for the Hoop instance itself to affect the hitboxes

Reply:
Made defensive copies again